sanic/signals.py

```diff
@@ -219,8 +217,11 @@ def add(  # type: ignore
         if not trigger:
             event = ".".join([*parts[:2], "<__trigger__>"])
 
-        handler.__requirements__ = condition  # type: ignore
-        handler.__trigger__ = trigger  # type: ignore
+        try:
```

Comment:
Can you throw in a code comment here about this being deprecated also? If nothing else, it will make it easier for me to locate it in the future.

```
Attaching __requirements__ and __trigger__ to the handler is deprecated and will be removed in v23.6.
```

Reply:
Done